context/context.go

```diff
@@ -25,8 +26,9 @@ const (
 type Config struct {
 	// Settings.
 
-	Context   netcontext.Context
-	SessionID string
+	Context     netcontext.Context
+	Expectation spec.Expectation
+	SessionID   string
 
 	// TODO we want to track the original input that was provided from the
```

Comment:
Now we lookup the original input using the information ID tracked in the context and send it to the input CLG referenced by the CLG tree ID tracked by the context